.gitmodules

```diff
@@ -7,9 +7,6 @@
 [submodule "packages/scallop-tk"]
 	path = packages/scallop-tk
 	url = https://github.com/mattdawkins/scallop-tk.git
-[submodule "packages/vibrant"]
-	path = packages/vibrant
-	url = https://github.com/Kitware/vibrant.git
 [submodule "packages/vivia"]
```

Comment:
It was already added in prior commit, ie, right now they are both checked in and one needs to be removed to prevent code duplication.